### PR TITLE
test: update other_test.go to correctly pass the prefix

### PR DESCRIPTION
**Pull request by MatthewLemmond**
### Description
Correctly pass the prefix to the test helper options 

### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content
test changes no release needed

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: tests/other_test.go [hunk 1]
```diff
@@ -16,9 +16,8 @@ func TestRunBasicExample(t *testing.T) {
 	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
 		Testing:      t,
 		TerraformDir: basicExampleTerraformDir,
-		Prefix:       "s2s-basic",
 		TerraformVars: map[string]interface{}{
-			"prefix": options.Prefix,
+			"prefix": "s2s-basic",
```
Comment (ocofaigh) [inline tests/other_test.go:20]: why are we hard coding a prefix here and not using the wrapper Prefix option to generate a unique for for every test?

Comment (MatthewLemmond) [reply tests/other_test.go]: I had forgotten about the random string being added by the wrapper, updated.

File: tests/other_test.go [hunk 2]
```diff
@@ -14,14 +14,14 @@ func TestRunBasicExample(t *testing.T) {
...
-		Testing:      t,
-		TerraformDir: basicExampleTerraformDir,
-		Prefix:       "s2s-basic",
-		TerraformVars: map[string]interface{}{
-			"prefix": options.Prefix,
-		},
+		Testing:       t,
+		TerraformDir:  basicExampleTerraformDir,
+		Prefix:        "s2s-basic",
+		TerraformVars: map[string]interface{}{},
```
Comment (ocofaigh) [inline tests/other_test.go:20]: Why keep `TerraformVars` here is we are not passing any to it?

Comment (MatthewLemmond) [reply tests/other_test.go]: moved out of options

**Comment by ocofaigh** [comment]
I think this is fine, can you confirm test ran fine locally for you?

**Comment by MatthewLemmond** [comment]
Yes, I confirmed that the test ran using this syntax and that it uses the random string on the prefix so it should now all be working for creating unique resource groups each run

**Pull request merged**